src/app/shared/utils/utils.ts

```diff
@@ -217,6 +219,17 @@ export class Util {
     return filterState;
   }
 
+  public static setPaginationParams(
+    params: PaginationParameters,
+    currentPage: PaginationElement,
+    itemsPerPage: number
+  ): PaginationParameters {
+    params.size = itemsPerPage;
+    params.from = (+currentPage.element - 1) * params.size;
+
+    return params;
```

Comment:
I see you return params but you don't assign anything in the components. I think it should look like workshopsParams = Util. Util.setPaginationParams